docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -172,7 +172,7 @@ EOF
 . Restore the databases from `.sql` files into the control plane MariaDB:
 +
 ----
-$ oc rsh mariadb-copy-data << EOF
+$ oc rsh -n {pod_ns} mariadb-copy-data << EOF
```

Comment:
the usage of -n namespace should not cause any difference in regular adoptions, since that's the default namespace users can use or omit that.
I'm adding and making the namespace a variables here to make as little changes as possible to the docs, so that when using ospdo it's explicit where we're running actions on.